pinniped-components/post-deploy/Makefile

```diff
@@ -10,6 +10,11 @@ GIT_VERSION ?= $(shell git describe --always --tags)
 GIT_REF_LONG = $(shell git rev-parse --verify HEAD)
 # Release version
 VERSION ?= $(GIT_VERSION)
+#Adding Support for GOPROXY and GOSUMDB
+GOPROXY ?= "https://proxy.golang.org,direct"
+GOSUMDB ?= "sum.golang.org"
+DISTROLESS_BASE_IMAGE ?= gcr.io/distroless/static:nonroot
```

Comment:
@vmware-tanzu/tkg-iam-owners : we're working on this changes to provide knobs in tooling to swap these values during productisation. Noticed pinniped component doesn't include `common.mk` in this Makefile (like rest of the components), so we're adding these vars directly in the Makefile. Checking if we should include `common.mk` in this Makefile to avoid this explicit addition ?